routes/teams.js

```diff
@@ -23,4 +24,101 @@ router.get('/team', function(req, res) {
     });
 });
 
+
+// when players endpoint is hit, call the API/DB using that team_id
+router.get('/players', function(req, res) {
+  temp = req.query.split('=');
```

Comment:
i think there is something maybe deeper in that object that will parse that out already? maybe not